pages/home.js

```diff
@@ -23,6 +25,10 @@ export default function Home(props) {
   const [createRoomError, setCreateRoomError] = useState(undefined)
   const [joinRoomError, setJoinRoomError] = useState(undefined)
 
+  //useRef doesn't notify the browser when mutating the current ref, so forceUpdate allows us to trigger a re-render instead of using useState and having a delay
+  const [, forceUpdate] = useReducer((x) => x + 1, 0)
+  const cardList = useRef([0, 1, 2, 3, 5, 8, 13, 20, 1001, 1002])
```

Comment:
I'm not convinced this a good strategy. Though I like the idea of introducing reducers but they would be better fit for the users/room state on the room page